static/js/chart/draw.ts

```diff
@@ -20,7 +20,7 @@ import { DataGroup, DataPoint } from "./base";
 
 const NUM_X_TICKS = 5;
 const NUM_Y_TICKS = 5;
-const MARGIN = { top: 20, right: 10, bottom: 30, left: 35, yAxis: 3 };
+const MARGIN = { top: 20, right: 10, bottom: 30, left: 35, yAxis: 3, sourceText: 8 };
```

Comment:
can make this a separate variable: SOURCE_FONT = 8;

this is for the margin at page level